workflow-templates/gradle.yml

```diff
@@ -30,11 +30,21 @@ jobs:
           key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle') }}
           restore-keys: ${{ runner.os }}-gradle
       - name: Wrap with specified version
-        run: gradle wrapper --gradle-version=${{ env.gradle_version }}
+        run: |
+          DIR=$PWD 
+          cd /tmp 
+          gradle wrapper --gradle-version=${{ env.gradle_version }}
+          mv .gradle gradle gradlew $DIR
         if: ${{ env.gradle_version != '' }}
+        shell: bash
       - name: Wrap without version
-        run: gradle wrapper
+        run: |
```

Comment:
In case the default installed version of Gradle is used, is this custom installation necessary?

Reply:
it is otherwise we cannot use ``gradlew`` in the following step

Reply:
I failed to find a failing build in these circumstances as it looks like all the `omero-*` workflows are overriding `gradle_version`.